src/model/evaluator.py

```diff
@@ -0,0 +1,17 @@
+"""
+Base class for an Evaluator
+"""
+
+from abc import ABC
+
+
+class Evaluator(ABC):
+    """
+    Base class for an Evaluator
+    """
+
+    def evaluate(self, input_file_path):
```

Comment:
```suggestion
    def evaluate(self, input_file_path: str):
```

Reply:
Please see the comment here: https://github.com/Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024/pull/14#discussion_r1385260273